packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -312,6 +312,9 @@ export class DebugConfigurationManager {
         }
         const { languageId } = widget.editor.document;
         const debuggers = await this.debug.getDebuggersForLanguage(languageId);
+        if (debuggers && debuggers.length === 0) {
```

Comment:
i think it is generic issue with `QuickPickServiceImpl` and should be fixed there instead

Reply:
Moved changes to the `quick-pick-service-impl.ts` and removed from `debug-configuration-manager.ts`